btrfs.h

```diff
@@ -15,22 +15,28 @@
 */
 
 #include <stddef.h>
+#include <stdint.h>
+#include <string.h>
```

Comment:
It looks like this is just here for memcpy -- why not move it into `btrfs.c` where it is used?

Reply:
done